lib/streamFromGeojson.js

```diff
@@ -18,10 +19,9 @@ const streamFromGeojson = (featureStream, filePath) => {
   const reader = fs.createReadStream(filePath);
   const parser = geojsonStream.parse();
 
-  reader.on('error', handleError);
+  reader.on('error', (err) => handleError(invalid(err.message)));
```

Comment:
@davidtheclark Now my mind is getting tangled up with "Is this technically invalid?"... :)

I'm considering just passing the error as-is and not passing it through `invalid()`